integreat_cms/cms/views/pages/page_tree_view.py

```diff
@@ -81,13 +85,33 @@ def get(self, request, *args, **kwargs):
                 "language_tree",
                 **{
                     "region_slug": region.slug,
-                }
+                },
             )
 
         if not request.user.has_perm("cms.change_page"):
-            messages.warning(
-                request, _("You don't have the permission to edit or create pages.")
-            )
+            access_granted_pages = Page.objects.filter(
+                Q(editors=request.user) | Q(publishers=request.user)
+            ).filter(region=request.region)
+            if request.user.organization:
+                access_granted_pages = access_granted_pages.union(
+                    Page.objects.filter(organization=request.user.organization)
+                )
+            if len(access_granted_pages) > 0:
+                page_list = ", ".join(
+                    f"<a href='{escape(translation.backend_edit_link)}' class='hover:underline'>{escape(translation.title)}</a>"
```

Comment:
With my PR #1862 now merged, messages containing links have to be formatted by [translate_link](https://github.com/digitalfabrik/integreat-cms/blob/a5006d1c1f69a5d24a9cc4d8b424f9bc79844f2b/integreat_cms/cms/utils/translation_utils.py#L26) (if it's only one link) or [format_html](https://docs.djangoproject.com/en/4.1/ref/utils/#django.utils.html.format_html)/[format_html_join](https://docs.djangoproject.com/en/4.1/ref/utils/#django.utils.html.format_html_join) (when having multiple links), otherwise they will be escaped.

Reply:
Ah ok now I got it why the link in the messsage did not work from the certain moment.
I reverted it back.